packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -314,6 +314,7 @@ module.exports = ctx => ({
               '/docs/reference/api/archive-myaccount/',
               '/docs/reference/api/myaccount-migration/',
               '/docs/reference/csi-delauth-hook/',
+              'd28m3l9ryqsunl.cloudfront.net/',
```

Comment:
Here we have paths listed for dev docs host https://github.com/okta/okta-developer-docs/blob/7f6a8a62d6d20472fdcb88ea4ef640d6b9e3c365/packages/%40okta/vuepress-site/.vuepress/config.js#L290
And it looks like there is no way to provide multiply hosts https://www.npmjs.com/package/vuepress-plugin-robots

Reply:
Removed it.